ROMFS/px4fmu_common/mixers/vtol_AERT.aux.mix

```diff
@@ -0,0 +1,30 @@
+Mixer for a horizontal format with X-Quad and tractor AERT plane configuration
```

Comment:
It's just meant to refer to the plane part. So instead of saying Fun Cub / Quad VTOL I say AERT / Quad. What did you mean with your previous one anyway?